README.md

```diff
@@ -260,38 +260,39 @@ The Trillian service is multi-tenanted &ndash; a single Trillian installation
 can support multiple Merkle trees in parallel, distinguished by their `TreeId`
 &ndash; and each tree operates in one of two modes:
 
- - **Log** mode: an append-only collection of items.
+ - **Log** mode: an append-only collection of items; this has two sub-modes:
+   - normal Log mode, where the Trillian service assigns sequence numbers to
+     new tree entries as they arrive
+   - 'preordered' Log mode, where the unique sequence number for entries in
+     the Merkle tree is externally specified
```

Comment:
Do we document anywhere *why* you would want to specify sequence numbers yourself, and any pros/cons?

Reply:
Not that I know of -- added issue #1367 as it doesn't belong here.

Reply:
Thanks :+1: